fifth_ex.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def add_a_vector_to_matrix(vector, matrix) -> None:
+    """
+    This function gets a vector and a matrix and creates a new matrix in which each row is each row
+    of the given matrix + the vector.
+    :param vector: the vector we want to add to each row of the matrix.
+    :param matrix: the matrix we want to add to each of it's rows the vector.
+    """
+    # Create a copy of given matrix
+    matrix_of_the_sum = np.empty_like(matrix)
+    num_of_rows, num_of_cols = matrix_of_the_sum.shape
+    for element in range(num_of_rows):
+        matrix_of_the_sum[element, :] = matrix[element, :] + vector
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html
This can all be done in one row and more efficiently (no need to create new matrix).